nvme_gw_cli.py

```diff
@@ -258,6 +317,28 @@ def create_listener(self, args):
         except Exception as error:
             self.logger.error(f"Failed to create listener: \n {error}")
 
+    @cli.cmd([
+        argument("-n", "--subnqn", help="Subsystem NQN", required=True),
+        argument("-a", "--traddr", help="NVMe host IP", required=True),
+        argument("-s", "--trsvcid", help="Port number", required=True),
+        argument("-t", "--trtype", help="Transport type", default="TCP"),
+        argument("-f", "--adrfam", help="Address family", default="ipv4"),
+    ])
+    def delete_listener(self, args):
+        """Deletes a listener at a particular TCP/IP address for a given subsystem."""
+
+        try:
+            delete_req = pb2.listener_delete_req(
+                nqn=args.subnqn,
+                trtype=args.trtype,
+                adrfam=args.adrfam,
+                traddr=args.traddr,
+                trsvcid=args.trsvcid,
+            )
+            ret = self.stub.nvmf_subsystem_remove_listener(delete_req)
+            self.logger.info(f"Deleted {args.traddr} from {args.subnqn}: {ret.status}")
+        except Exception as error:
+            self.logger.error(f"Failed to create listener: \n {error}")
```

Comment:
Same here for the listener:
```suggestion
            self.logger.error(f"Failed to delete listener: \n {error}")
```